packages/toolkit/src/view/knowledge/components/tabs/ImageTab.tsx

```diff
@@ -0,0 +1,176 @@
+                  {Array.from({ length: 6 }, (_, i) => (
+                    <div
+                      key={i}
+                      className="flex flex-col gap-y-2.5 rounded-md border border-semantic-bg-line bg-semantic-bg-primary p-2.5"
+                    >
+                      <div className="flex flex-col gap-y-2.5 p-2.5">
+                        <div className="flex items-center justify-between">
+                          <Tag size="sm" variant="default" className="!rounded">
+                            <span className="mr-1.5 product-body-text-3-medium">
+                              {String(i + 1).padStart(3, "0")}
+                            </span>
+                          </Tag>
+                          <div className="flex items-center gap-1">
+                            <span className=" uppercase product-label-label-1">
+                              {item.status ? "Retrievable" : "Unretrievable"}
+                            </span>
+                            <Switch
+                              checked={item.status}
+                              onCheckedChange={() => {}}
+                              className=""
+                            ></Switch>
+                          </div>
+                        </div>
+                        <div className="h-px w-full bg-semantic-bg-line" />
+                        <p className="text-semantic-fg-secondary-alt-secondary line-clamp-3 product-body-text-2-regular">
+                          Lorem ipsum dolor sit amet, consectetur adipiscing
+                          elit. Fusce ac justo vitae sem ultricies consectetur.
+                          Import your own text data or write in real-time via
+                          Webhook to enhance your LLM context. Effortlessly
+                          build a comprehensive catalog.
+                        </p>
+                        <div className="flex items-center justify-end gap-1">
+                          <Tag size="sm" variant="default" className="!rounded">
+                            <Icons.Type01 className="mr-1 h-2.5 w-2.5 stroke-semantic-fg-primary" />
+                            <span className="product-body-text-3-medium">
+                              2,235
+                            </span>
+                          </Tag>
+                          <Tag size="sm" variant="default" className="!rounded">
+                            <Icons.Type01 className="mr-1 h-2.5 w-2.5 stroke-semantic-fg-primary" />
+                            <span className="product-body-text-3-medium">
+                              0
+                            </span>
+                          </Tag>
+                          <Tag size="sm" variant="default" className="!rounded">
+                            <Icons.Type01 className="mr-1 h-2.5 w-2.5 stroke-semantic-fg-primary" />
+                            <span className="product-body-text-3-medium">
+                              2cfefe7706...
```

Comment:
@thewbuk Is this intended?

Reply:
Again another example of component waiting for V2